sdk/search/Azure.Search.Documents/README.md

```diff
@@ -319,7 +319,7 @@ SearchOptions options = new SearchOptions
     // Filter to only Rating greater than or equal our preference
     Filter = SearchFilter.Create($"Rating ge {stars}"),
     Size = 5, // Take only 5 results
-   // OrderBy = { "Rating desc" } // Sort by Rating from high to low
+    OrderBy = { "Rating desc" } // Sort by Rating from high to low
```

Comment:
If you're adding this back in, shouldn't you re-record? That said, as long as you're not asserting the order, I guess the already-recorded session doesn't really matter.

Reply:
I didn't change the recordings when I commented out this line for nightly testing purposes. 

[Here's](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2109985&view=logs&j=d76cd559-fb8e-56d0-cec0-e45f7cdf7dbf&t=be861a6a-e72b-540a-ce94-c251e2053297) the recording for OrderBy.